grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractServerCall.java

```diff
@@ -266,15 +266,16 @@ final void closeListener(Status newStatus, boolean completed, boolean setRespons
                 ctx.logBuilder().requestContent(GrpcLogUtil.rpcRequest(method, simpleMethodName), null);
             }
             if (setResponseContent) {
-                ctx.logBuilder().responseContent(GrpcLogUtil.rpcResponse(newStatus, firstResponse()), null);
+                ctx.logBuilder().responseContent(GrpcLogUtil.rpcResponse(newStatus, firstResponse(),
```

Comment:
How about inlining `GrpcLogUtil.rpcResponse(...)` method?
I'm asking because we create `StatusException` multiple times for the same metadata.
When creating `StatusException`, it calls `fillInStackTrace` which is expensive so I think we should create it only once if it's possible.

Reply:
It wouldn't be easy to statically check if `.asException()` is called before.
To minimize calling `fillStackTrace`, we can introduce a new class that can cache a `StatusException` internally.

```java
class StatusAndMetadata {
  private final Status status;
  private final Metadata metadata;

  @Nullable
  private StatusException statusException;

  StatusException asStatusException() {
    if (statusExcepion != null) {
      return statusException;
    }
    
    return statusException = status.asStatusException(metadata);
  }
}
```

However, it seems out of the scope of this PR. What do you think of handling the performance issue separately?

Reply:
That's a good idea. Let's handle it in the follow-up PR. 😉